lib/rules/jsx-max-props-per-line.js

```diff
@@ -24,8 +24,17 @@ module.exports = {
       type: 'object',
       properties: {
         maximum: {
-          type: 'integer',
-          minimum: 1
+          type: 'object',
+          properties: {
```

Comment:
this is a breaking change unless it uses `anyOf` or `oneOf` to allow both an integer *and* an object.